array/DNAm/analysis/EWAS/summariseWithinCTDMPs.r

```diff
@@ -0,0 +1,272 @@
+        xlab = "With CETYGO", ylab = "Without CETYGO", main = cellTypes[i], col = "darkgrey")
+        abline(a = 0, b = 1)
+        plot(-log10(res[[i]][,"CCModel_SCZ_P"]), -log10(res[[i]][,"NullModel_SCZ_P"]), pch = 16, 
+        xlab = "With cell composition", ylab = "Without cell composition", main = cellTypes[i], col = "darkgrey")
+        abline(a = 0, b = 1)
+    } else {
+        plot(-log10(res[[i]][,"FullModel_SCZ_P"]), -log10(res[[i]][,"NullModel_SCZ_P"]), pch = 16, 
+        xlab = "With CETYGO", ylab = "Without CETYGO", main = cellTypes[i], col = "darkgrey")
+        abline(a = 0, b = 1)
+    }
+}
+dev.off()
+
+# Manhattan plots
+
+png(file.path(resPath, "Plots", "ManhattanPlotsWithinCTLM.png"), width = 1400, height = 1000, res = 200)
+par(mfrow = c(3,1))
+par(mar = c(4,4,0.5,0.5))
+par(mgp = c(2,0.75,0))
+for(i in 1:3){
+    res[[i]][,"chrm"][which(res[[i]][,"chrm"] == "X")]<-"23"
+    res[[i]][,"chrm"]<-as.numeric(as.character(res[[i]][,"chrm"]))
+
+    resMan<-data.frame("SNP" = rownames(res[[i]]), "P" = res[[i]][,"FullModel_SCZ_P"], "CHR" = res[[i]][,"chrm"],"BP" = res[[i]][,"start"])
+    resMan<-na.omit(resMan)
+    manhattan(resMan, genomewideline = -log10(9e-8), suggestiveline = -log10(1e-5), main = cellTypes[i])
+} 
+dev.off()
+
+
+#----------------------------------------------------------------------#
+# IDENTIFY DMPS
+#----------------------------------------------------------------------#
+
+## count DMPs at diff thresholds
+nDMPs<-matrix(ncol = 9, nrow = 4)
+colnames(nDMPs)<-outer(c("FullModel", "CCModel", "NullModel"),  cellTypes, paste, sep = ":")
+rowIndex<-1
+for(thres in c(9e-8, 1e-7, 1e-6, 1e-5)){
+    for(i in 1:3){
+        nDMPs[rowIndex,(i*3)-2]<-length(which(res[[i]][,"FullModel_SCZ_P"] < thres))
+        if("CCModel_SCZ_P" %in% colnames(res[[i]])){
+            nDMPs[rowIndex,(i*3)-1]<-length(which(res[[i]][,"CCModel_SCZ_P"] < thres))
+        }
+        nDMPs[rowIndex,(i*3)]<-length(which(res[[i]][,"NullModel_SCZ_P"] < thres))
+    }
+    rowIndex<-rowIndex+1
+}
+cbind()
+
+write.csv(nDMPs, file.path(resPath, "Tables", "DMPCountsPerCelltypeAcrossWithinCTModels.csv"))
+
+## COMPARE DMPS ACROSS CELL TYPES
+thres<-1e-5
+
+dmpList<-NULL
+for(i in 1:3){
+    dmpList<-rbind(dmpList, cbind(rownames(res[[i]])[which(res[[i]][,"FullModel_SCZ_P"] < thres)], cellTypes[i]))
+}
+colnames(dmpList)<-c("ProbeID", "DiscoveryCellType")
+
+
+
+dmpRes<-cbind(dmpList, res[[1]][dmpList[,1], c("FullModel_SCZ_P", "FullModel_SCZ_coeff", "FullModel_SCZ_SE")],
+res[[2]][dmpList[,1], c("FullModel_SCZ_P", "FullModel_SCZ_coeff", "FullModel_SCZ_SE")],
+res[[3]][dmpList[,1], c("FullModel_SCZ_P", "FullModel_SCZ_coeff", "FullModel_SCZ_SE")])
+colnames(dmpRes)<-c("ProbeID", "DiscoveryCellType", outer(c("P", "Coeff", "SE"), cellTypes, paste, sep = ":"))
+
+
+
+diffLong<-pivot_longer(data.frame(dmpRes[,c(2,4,7,10)]), cols = gsub("\\+|-", "\\.", paste("Coeff", cellTypes, sep = ".")))
```

Comment:
Possible typo here: Did you mean to put `sep = ":"` here instead of `sep = "."`?